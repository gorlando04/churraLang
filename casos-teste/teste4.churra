amigo "X" contribui 80
amigo "Y" contribui 120
item carne "alcatra" preco_kg 50 quantidade_kg 2
rateio igual