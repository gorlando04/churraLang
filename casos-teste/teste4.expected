Total do churrasco: R$100.00
X deve pagar: R$50.00
Y deve pagar: R$50.00